tests/sysfs/procSysKernelPaniconoops.bats

```diff
@@ -77,25 +74,3 @@ function teardown() {
   [ "$output" = $KERNEL_PANIC_OOPS_DEFAULT_VAL ]
 }
 
-# Write an invalid/unsupported value (< 0 && > 1).
```

Comment:
I think we talked about this one, but can't remember, why is this an invalid test?

Reply:
The test is invalid because the kernel does allow higher values to be written (i.e., it does not reject them, even though their behavior is not defined).